Cargo.toml

```diff
@@ -12,3 +12,7 @@ members = [
 
 [profile.release]
 lto = "thin"
+
+# FIXME(the-wondersmith): For end-to-end testing before merging upstream changes.
+[patch.crates-io]
+k8s-gateway-api = { git = "https://github.com/the-wondersmith/k8s-gateway-api-rs", branch = "feat-grpcroute-support" }
```

Comment:
This isn't needed in this branch, right? If we remove it (and regen lock) we'll be able to merge this while we vet the rust work.

Reply:
Done and done 😁